### DISCUSSION TITLE
All deploys fail with error "Username and password required"

### DISCUSSION THREAD

**Discussion by aw632** [alshedivat/al-folio#765] [Q&A] [answered]
Hi,

Whenever I deploy, my deploy named `Docker Image CI` fails. Attached are the logs. It gives me the error "username and password required". I couldn't find any details about username and password in the readme, is this something I should be setting..?
[logs_7.zip](https://github.com/alshedivat/al-folio/files/9116619/logs_7.zip)

**Comment by rohandebsarkar** [comment, marked as answer]
I think this is because of the `docker-deploy.yml` script.

@pourmand1376, do you have a solution for this? How about only deploying to docker on tag releases?

**Reply by alshedivat** [reply]
I think we should definitely disable docker deploys on fork repos. do you guys know if github actions allow detecting which repo tries to run the action? or maybe checking the owner of the repo is sufficient?

**Reply by pourmand1376** [reply]
I have found a solution [here](https://github.community/t/have-github-action-only-run-on-master-repo-and-not-on-forks/140840/18) and also added a PR #767. 

This is basically a workaround since Github doesn't officially support this thing. But I've tested and it workes perfectly. 

```
    if: github.repository_owner == 'alshedivat'
```

![image](https://user-images.githubusercontent.com/32064808/179180898-9d5414e7-ab34-4fff-b92b-a205b47665b3.png)
